sdk/src/main/scala/com/horizen/api/http/ApiResponseUtil.scala

```diff
@@ -18,28 +18,16 @@ object ApiResponseUtil {
         e.exception.asScala match {
           case Some(thr) =>
             val msg = thr.getMessage
-            if (msg != null && !msg.isEmpty)
+            if (msg != null && msg.nonEmpty)
               SidechainApiResponse(SerializationUtil.serializeErrorWithResult(e.code, e.description, msg))
             else SidechainApiResponse(SerializationUtil.serializeErrorWithResult(e.code, e.description, ""))
           case None => SidechainApiResponse(SerializationUtil.serializeErrorWithResult(e.code, e.description, ""))
         }
     }
   }
 
-  def toResponseWithoutResultWrapper(response: ApiResponse): Route = {
-    response match {
-      case _: SuccessResponse => SidechainApiResponse(SerializationUtil.serialize(response))
-      case e: ErrorResponse =>
-        e.exception.asScala match {
-          case Some(thr) =>
-            val msg = thr.getMessage
-            if (msg != null && !msg.isEmpty)
-              SidechainApiResponse(SerializationUtil.serialize(e.code, e.description, msg))
-            else SidechainApiResponse(SerializationUtil.serialize(e.code, e.description, ""))
-          case None => SidechainApiResponse(SerializationUtil.serialize(e.code, e.description, ""))
-        }
-    }
-  }
+  def toResponseWithoutResultWrapper(response: ApiResponse): Route =
+    SidechainApiResponse(SerializationUtil.serialize(response))
```

Comment:
why do we have this change?
I can harm the original behavior of API responses.
pls, doublecheck

Reply:
I assume that it was changed here, because it is not used anywhere, but we should move it to AccountEthRpcRoute